src/components/collection/Summary.tsx

```diff
@@ -0,0 +1,179 @@
+import truncate from 'lodash.truncate';
+
+import React, { useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Typography } from '@mui/material';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+
+import { Category, CategoryType, ItemCategory } from '@graasp/sdk';
+import { ItemRecord, MemberRecord } from '@graasp/sdk/dist/frontend/types';
```

Comment:
I don't think these should be imported from `dist`, but `@graasp/sdk/frontend` @spaenleh ?

Reply:
Yes correct import is from `@graasp/sdk/frontend` but i have seen that some IDEs do not suggest that path easily ...